yt_dlp/extractor/rule34video.py

```diff
@@ -50,12 +52,14 @@ def _real_extract(self, url):
         title = self._html_extract_title(webpage)
         thumbnail = self._html_search_regex(r'preview_url:\s+\'([^\']+)\'', webpage, 'thumbnail', default=None)
         duration = self._html_search_regex(r'"icon-clock"></i>\s+<span>((?:\d+:?)+)', webpage, 'duration', default=None)
+        tags = re.findall(r'<a class="tag_item"\s+href="https://rule34video\.com/tags/\d+/".*>(?P<tag>.*)</a>', webpage)
```

Comment:
```suggestion
        tags = re.findall(r'<a class="tag_item"\s+href="https://rule34video\.com/tags/\d+/"[^>]*>(?P<tag>[^>]*)</a>', webpage)
```

Also, do we really need to check the URL? Isn't the class enough?

```py
            'tags': re.findall(r'<a class="tag_item"[^>]*>(?P<tag>[^>]*)</a>', webpage)
```

Reply:
Unfortunately yes, otherwise other elements also get included that aren't tags (e.g. `['MP4 1080p', 'MP4 720p', 'MP4 480p', 'MP4 360p']` are included when the URL check is not there).